justfile

```diff
@@ -14,6 +16,47 @@ init-db: generate-db-certs
 generate-db-certs:
     cd backend && TEST_CERT_DIR_NAME=test_certs scripts/generate_test_certs.sh
 
+# === App ===
+
+app_dir := "prototype"
+app_lib_name := "applogic"
+app_rust_base_dir := app_lib_name
+
 # generate Rust and Dart flutter bridge files
-generate-flutter-ffi:
-    cd prototype && make frb-generate
+frb-generate:
+    rm -f {{app_rust_base_dir}}/src/frb_*.rs
+    touch {{app_rust_base_dir}}/src/frb_generated.rs
+    mkdir -p {{app_dir}}/lib/core
+    rm -Rf {{app_dir}}/lib/core/*
+    cd {{app_dir}} && flutter pub get
+    cd {{app_dir}} && flutter_rust_bridge_codegen generate
+    cd {{app_dir}} && flutter clean
+
+# integrate the Flutter Rust bridge
+frb-integrate:
+	cd {{app_dir}} && mv flutter_rust_bridge.yaml flutter_rust_bridge.yaml.tmp
```

Comment:
Leaving here for reference: when https://github.com/casey/just/pull/2438 is released, we can remove all `cd`s from here.